src/main/java/com/ericsson/ei/handlers/ObjectHandler.java

```diff
@@ -278,9 +281,9 @@ public String lockDocument(String id) {
      */
     public int getTtl() {
         int ttl = 0;
-        if (ttlValue != null && !ttlValue.isEmpty()) {
+        if (aggregationsTtl != null && !aggregationsTtl.isEmpty()) {
```

Comment:
I saw StringUtils being used in other places so maybe you could use StringUtils.isEmpty here.

Reply:
Ok, will fix that